sway-lib-std/src/math.sw

```diff
@@ -214,7 +214,14 @@ impl BinaryLogarithm for u8 {
 impl BinaryLogarithm for u256 {
     fn log2(self) -> Self {
         use ::assert::*;
-        assert(self != 0);
+        use ::registers::flags;
+
+        // If panic on unsafe math is enabled, only then revert
+        if flags() & 0b00000000_00000000_00000000_00000000_00000000_00000000_00000000_00000001 == 0 {
```

Comment:
Are you able to import the const so this is more readable?

Reply:
The const is not pub and i didnt want to introduce changes to that library in this pr. Should i make it pub here or is it fine if its fixed in a different PR?

Reply:
I think it's okay to change it here